apps/omg_eth/config/dev.exs

```diff
@@ -1,8 +1 @@
 use Mix.Config
-
```

Comment:
overrides here are useful. When working in `:dev` the shorter exit period (maybe 10 minutes, not 4 though) will be a good default.

So will be the `recv_timeout` in `ethereumex` - this is for unlocking a lot of accounts on the eth node, which is a useful piece of the flow when running demos/experiments etc